src/allotropy/allotrope/models/light_obscuration_rec_2021_12_light_obscuration.py

```diff
@@ -0,0 +1,66 @@
+# generated by datamodel-codegen:
+#   filename:  light-obscuration.json
+#   timestamp: 2023-12-04T14:21:51+00:00
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Optional, Union
+
+from allotropy.allotrope.models.shared.definitions.custom import (
+    TQuantityValueCountsPerMilliliter,
+    TQuantityValueMicrometer,
+    TQuantityValueMilliliter,
+    TQuantityValueUnitless,
+)
+from allotropy.allotrope.models.shared.definitions.definitions import (
+    TBooleanValue,
+    TDateTimeValue,
+    TStringValue,
+)
+
+
+@dataclass
+class RepetitionSetting:
```

Comment:
This value is `tIntValue`. Seems like this is something we should add to `definitions` (like `tStringValue`)

Reply:
Done.